src/typings/place.d.ts

```diff
@@ -1,4 +1,6 @@
 declare interface Place {
+  stepLabel?: string
```

Comment:
I would name them so that we know what they are used for just by reading the names:
stepAriaLabel
actionAriaLabel